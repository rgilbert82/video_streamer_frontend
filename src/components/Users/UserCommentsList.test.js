import React                from 'react';
import { shallow }          from 'enzyme';
import { UserCommentsList } from '.';

describe(UserCommentsList, () => {
  let component;

  beforeEach(() => {
    component = shallow(<UserCommentsList comments={[]} />);
  });

  it('renders properly', () => {
    expect(component).toMatchSnapshot();
  });
});
