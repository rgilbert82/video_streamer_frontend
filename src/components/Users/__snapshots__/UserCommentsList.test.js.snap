// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`class UserCommentsList extends _react2.default.Component {
  render() {
    const comments = this.props.comments.map(commentObj => {
      return (
        _react2.default.createElement('li', { key: commentObj.comment.id, __source: { fileName: _jsxFileName, lineNumber: 8 }, __self: this },
          _react2.default.createElement(_.UserCommentListItem, { commentObj: commentObj, __source: { fileName: _jsxFileName, lineNumber: 9 }, __self: this })));


    });

    return (
      _react2.default.createElement('div', { __source: { fileName: _jsxFileName, lineNumber: 15 }, __self: this },
        _react2.default.createElement('h3', { __source: { fileName: _jsxFileName, lineNumber: 16 }, __self: this }, 'Chat Comments'),
        _react2.default.createElement('ul', { __source: { fileName: _jsxFileName, lineNumber: 17 }, __self: this },
          comments)));



  }} renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <UserCommentsList
    comments={Array []}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3>
          Chat Comments
</h3>,
        <ul>
          
</ul>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Chat Comments",
        },
        "ref": null,
        "rendered": "Chat Comments",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [],
        },
        "ref": null,
        "rendered": Array [],
        "type": "ul",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Chat Comments
</h3>,
          <ul>
            
</ul>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Chat Comments",
          },
          "ref": null,
          "rendered": "Chat Comments",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [],
          },
          "ref": null,
          "rendered": Array [],
          "type": "ul",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
